gateway-service-knoxtoken/src/test/java/org/apache/knox/gateway/service/knoxtoken/TokenServiceResourceTest.java

```diff
@@ -679,6 +680,40 @@ public void testTokenRenewal_Enabled_WithRenewersWithValidSubject() throws Excep
     validateSuccessfulRenewalResponse(renewalResponse);
   }
 
+  @Test
+  public void testTokenRenewal_Enabled_WithDefaultMaxTokenLifetime() throws Exception {
+    final String caller = "yarn";
+
+    // Max lifetime duration is 10ms
+    Pair<TestTokenStateService, Response> testResult =
+                  doTestTokenRenewal(true, caller, null, createTestSubject(caller));
+
+    TestTokenStateService tss = testResult.getLeft();
+    assertEquals(1, tss.issueTimes.size());
+    String token = tss.issueTimes.keySet().iterator().next();
+
+    // Verify that the configured max lifetime was honored
+    assertEquals(tss.getDefaultMaxLifetimeDuration(), tss.getMaxLifetime(token) - tss.getIssueTime(token));
+  }
+
+
+  @Test
+  public void testTokenRenewal_Enabled_WithConfigurableMaxTokenLifetime() throws Exception {
```

Comment:
Do we need the underscores in the method name (see in another new method above too)?

Reply:
It's test code, and it helps me to quickly distinguish test methods that vary only slightly in name. So, are the needed? No. Does it matter? It's test code.